detection_rules/rule.py

```diff
@@ -1084,6 +1095,9 @@ def get_packaged_integrations(cls, data: QueryRuleData, meta: RuleMeta,
         packaged_integrations = []
         datasets = set()
 
+        if data.type == "esql":
+            return packaged_integrations
```

Comment:
We are skipping for now, but we can probably get these from the `integrations` tag (assuming we won't have the `ast`.